.github/workflows/lint-bpf-checks.yaml

```diff
@@ -120,4 +124,27 @@ jobs:
           fetch-depth: 0
       - name: Run BPF tests
         run: |
-          make run_bpf_tests || (echo "Run 'make run_bpf_tests' locally to investigate failures"; exit 1)
+          make run_bpf_tests \
+              LOG_CODEOWNERS=1 \
+              JUNIT_PATH="../../test/${{ env.job_name }}.xml" \
+          || (echo "Run 'make run_bpf_tests' locally to investigate failures"; exit 1)
+      - name: Fetch JUnits
+        shell: bash
+        run: |
+          mkdir -p cilium-junits
+          cd test/
+          # junit_filename needs to be the same as the Job Name presented on the
+          # GH web UI - In the Summary page of a workflow run, left column
+          # "Jobs" - so that we can map the junit file to the right job - step
+          # pair on datastudio.
+          junit_filename="${{ env.job_name }}.xml"
+          for filename in *.xml; do cp "${filename}" "../cilium-junits/${junit_filename}"; done;
+      - name: Upload JUnits [junit]
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: cilium-junits-${{ env.job_name }}
```

Comment:
```suggestion
          name: cilium-junits
```

Reply:
Alternative is to introduce another job for `merge-upload` similar to the other junit gathering workflows, but I don't anticipate gathering junits from the other jobs in this workflow so that doesn't seem necessary.